<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgSoftOptions</class>
 <widget class="QWidget" name="WdgSoftOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>280</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>255</horstretch>
    <verstretch>60</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>385</width>
    <height>280</height>
   </size>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="diameterLayout">
    <item>
     <widget class="QLabel" name="radiusLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>diameter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KDoubleNumInput" name="diameterSpinBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="value">
       <double>20.000000000000000</double>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="sliderEnabled">
       <bool>true</bool>
      </property>
      <property name="exponentRatio">
       <double>1.500000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>108</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="tmpLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="radiusLabel_2">
      <property name="text">
       <string>Begin</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="startSPBox">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="radiusLabel_3">
      <property name="text">
       <string>End</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="endSPBox">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>361</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="spacingLayout">
    <item>
     <widget class="QLabel" name="spacingLbl">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>spacing</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KDoubleNumInput" name="spacingSPBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="value">
       <double>0.200000000000000</double>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="sliderEnabled">
       <bool>true</bool>
      </property>
      <property name="exponentRatio">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="spacingLbl_2">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>90</y>
     <width>49</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>sigma</string>
   </property>
  </widget>
  <widget class="KDoubleNumInput" name="sigmaSPBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>306</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="value">
    <double>2.000000000000000</double>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.200000000000000</double>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="sliderEnabled">
    <bool>true</bool>
   </property>
   <property name="exponentRatio">
    <double>1.500000000000000</double>
   </property>
  </widget>
  <widget class="KDoubleNumInput" name="flowSPBox">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>130</y>
     <width>306</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="value">
    <double>100.000000000000000</double>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="sliderEnabled">
    <bool>true</bool>
   </property>
   <property name="exponentRatio">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="flowLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>49</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>flow</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
